pegasus/rake/seed.rake

```diff
@@ -166,14 +167,20 @@ class GSheetToCsv
       puts "Error on file: #{@gsheet_path}, #{e}"
       throw e
     end
-    if @exclude_columns.empty?
+    if @include_columns.empty? && @exclude_columns.empty?
```

Comment:
Is there benefit to having the if-else? The execution will be more efficient (not that we care?), at the cost of adding code complexity.